presentation/src/main/java/com/anytypeio/anytype/presentation/sets/ObjectSetViewModel.kt

```diff
@@ -1721,12 +1736,14 @@ class ObjectSetViewModel(
     private fun proceedWithEditingTemplate() {
         val template = templatesWidgetState.value.moreMenuTemplate ?: return
         templatesWidgetState.value = templatesWidgetState.value.copy(
+            showWidget = false,
             isEditing = false,
             isMoreMenuVisible = false,
             moreMenuTemplate = null
         )
         viewModelScope.launch {
-            proceedWithOpeningObject(template.id)
+            delay(200)
```

Comment:
why do we need this delay here?

Reply:
we need to wait until modal screen with templates will close